include/aspect/particle/property/interface.h

```diff
@@ -259,7 +259,17 @@ namespace aspect
          * integration of solution properties or time varying particle
          * properties are used while solving the model problem.
          */
-        update_time_step
+        update_time_step,
+        /**
+         * Update the particles at every particle advection substep. The substeps are
+         * defined by the particle integrator which include an euler, a RK2 and a RK4
```

Comment:
R is one of those weird characters in English that is a consonant, but is treated like a vowel for the purpose of a/an (because it is pronounced Arrrrr, or something like that :-)).

```suggestion
         * defined by the particle integrator which include an Euler, an RK2 and an RK4
```